app/_locales/en/messages.json

```diff
@@ -362,9 +366,13 @@
     "message": "Approve spend limit"
   },
   "approveAllTokensTitle": {
-    "message": "Allow access to and transfer of all your $1?",
```

Comment:
I also think this of should remain, just grammatically speaking. @coreyjanssen?

Reply:
@adonesky1 Done [here](https://github.com/MetaMask/metamask-extension/pull/17992/commits/eb4ca37c7710f97deef4985ff0fe52d30ac19ddb).